code/datums/traits/good.dm

```diff
@@ -108,3 +108,16 @@
 	mob_trait = TRAIT_VORACIOUS
 	gain_text = "<span class='notice'>You feel HONGRY.</span>"
 	lose_text = "<span class='danger'>You no longer feel HONGRY.</span>"
+/datum/trait/hlumi_eyes
+	name = "High Luminosity Eyes"
+	desc = "You decided to upgrade your weak fleshy eyes with these cool cybernetic ones, your vision is augumented!"
+	value = 1
+
+/datum/trait/hlumi_eyes/on_spawn()
+	var/mob/living/carbon/human/H = trait_holder
+	var/obj/item/organ/eyes/robotic/glow = new
+	glow.Insert(H, drop_if_replaced = FALSE)
+	H.regenerate_icons()
+
+/datum/trait/hlumi_eyes/post_add()
+	to_chat(trait_holder, "<span class='boldannounce'>Your eyes were replaced with a High Luminosity Eyes. You can change their color at will.</span>")
```

Comment:
"a High Luminosity Eyes" isn't a proper noun, I'd say replace it with something like `high-luminosity ones` to change it to `Your eyes were replaced with high-luminosity ones.` Boldannounce is also a weird span here because it's used for warnings. I'd say replace with with something like `boldnotice` because it's a positive change, not a negative one like Prosthetic Limb.